sigs.yaml

```
@@ -1098,36 +1098,24 @@ sigs:
       slack: sig-docs
       mailing_list: https://groups.google.com/forum/#!forum/kubernetes-sig-docs
       teams:
+      - name: kubernetes-blog
+        description: Kubernetes blog
       - name: sig-docs-maintainers
         description: Documentation maintainers
       - name: sig-docs-pr-reviews
         description: Documentation PR reviews
       - name: sig-docs-en-owners
         description: English content (default)
-      - name: sig-docs-es-owners
-        description: Spanish content
-      - name: sig-docs-de-owners
-        description: German content
-      - name: sig-docs-fr-owners
-        description: French content
-      - name: sig-docs-it-owners
-        description: Italian content
-      - name: sig-docs-ja-owners
-        description: Japanese content
-      - name: sig-docs-ko-owners
-        description: Korean content
-      - name: sig-docs-zh-owners
-        description: Chinese content
     subprojects:
     - name: reference-docs
       owners:
       - https://raw.githubusercontent.com/kubernetes-incubator/reference-docs/master/OWNERS
     - name: website
       owners:
       - https://raw.githubusercontent.com/kubernetes/website/master/OWNERS
-    - name: website-metadata
+    - name: Kubernetes-blog
       owners:
-      - https://raw.githubusercontent.com/kubernetes-sigs/website-metadata/master/OWNERS
+      - https://github.com/kubernetes/website/blob/master/content/en/blog/OWNERS
```

we need this to be the raw link:
```suggestion
      - https://raw.githubusercontent.com/kubernetes/website/master/content/en/blog/OWNERS
```